lua/hover/config.lua

```diff
@@ -1,14 +1,18 @@
 local M = {}
 
+---@class HoverConfig
 local default_config = {
   preview_opts = {
     border = 'single'
   },
+  use_preview_window = true,
```

Comment:
Not too happy with this config, it's more of a placeholder for now. Any ideas? I was first thinking of something along the lines of

```lua
{
    focus_opts = {
        style = "focus" | "previewwindow" | ...
    }
}
```

Reply:
This is fine for now. Just make sure to add a comment explaining the behaviour. I forgot `K` focussed the window, opening a preview instead is reasonable since `<C-w>w` usually focus's the hover.